### DISCUSSION TITLE
Support for external articles

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#289] [Ideas]
It would be nice to have support for external articles, something like the blowfish theme has.

Example [here](https://n9o.xyz/) (posts about docker).

It uses the parameter externalUrl in the [frontmatter](https://github.com/nunocoracao/n9o.xyz/blob/main/content/posts/202309-docker-423/index.md?plain=1).

This can be used not only for plain external articles, but also for sponsored posts or even ads.

**Comment by razonyang** [comment]
LGTM, closed and track on hbstack/blog#760

**Comment by razonyang** [comment]
Available since [v0.27.0](https://github.com/hbstack/blog/releases/tag/v0.27.0).
